config/locales/de.yml

```diff
@@ -569,37 +569,37 @@ de:
   simple_form:
     placeholders:
       job:
-        company: 'e.g. codebar'
-        company_website: 'e.g. https://codebar.io'
-        title: 'e.g. Internship'
-        description: 'Use text or markdown for the job description'
-        location: 'e.g. London or Berlin'
+        company: 'z.B. codebar'
+        company_website: 'z.B. https://codebar.io'
+        title: 'z.B. Praktikum'
+        description: 'Verwenden Sie Text oder Markdown für die Stellenbeschreibung'
+        location: 'z.B. London order Berlin'
     hints:
       job:
-        remote: Only check if the role is fully remote only
-        salary: Annual pay before tax, without commas or decimal points
+        remote: Nur prüfen, ob die Rolle vollständig ferngesteuert ist
+        salary: Jahresarbeitsentgelt vor Steuern, ohne Komma oder Dezimalzeichen
   activerecord:
     attributes:
       job:
-        title: Title
+        title: Titel
         company: Name
-        company_address: Address
-        company_postcode: Postcode
+        company_address: Adresse
+        company_postcode: Postleitzahl
         company_website: Website
-        email: Contact email
-        link_to_job: Link to job post
-        expiry_date: Application closing date
+        email: Kontakt E-Mail
+        link_to_job: Link zur Stellenausschreibung
+        expiry_date: Einsendeschluss
   donation_platforms:
-    title: "How to support us"
+    title: "Wie Sie uns unterstützen können"
     donation_form:
-      title: "Use the donation page"
-      text: "Support us with one-time or regular donation through the donation form on our website."
+      title: "Verwenden Sie die Spendenseite"
+      text: "Unterstützen Sie uns mit einer einmaligen oder regelmäßigen Spende über das Spendenformular auf unserer Website."
     amazon_smile:
       title: "Amazon Smile"
-      text: "Pick us as your charitable organisation while shopping on Amazon. This allows us to receive 0.5% of the net purchase price of eligible purchases as a donation."
+      text: "Wählen Sie beim Einkaufen auf Amazon uns als wohltätige Organisation aus. Auf diese Weise erhalten wir 0,5 % des Nettokaufpreises der in Frage kommenden Einkäufe als Spende."
```

Comment:
`Wählen Sie uns beim Einkaufen auf Amazon als gemeinnützige Organisation aus. So erhalten wir 0,5 % des Nettokaufpreises der in Frage kommenden Einkäufe als Spende.`